packages/http-signature-utils/src/utils/signatures.ts

```diff
@@ -17,13 +16,6 @@ export interface SignatureHeaders {
   'Signature-Input': string
 }
 
-const createSigner = (privateKey: KeyLike): Signer => {
-  const signer = async (data: BinaryLike) =>
-    sign(null, data as Buffer, privateKey)
-  signer.alg = 'ed25519' as Algorithm
-  return signer
-}
-
 export const createSignatureHeaders = async ({
```

Comment:
should we add some basic tests for this method, to show that it generates correctly named values?

Reply:
To ensure I have a complete understanding: are the tests you are talking about, designed to verify that the `Signature-Input` includes the `alg`, `created`, and `keyId` parameters after generating?

Reply:
Yup, something basic like that.